share/revng/abi/SystemV_x86_regparm_1.yml

```diff
@@ -13,7 +13,7 @@ CalleeIsResponsibleForStackCleanup: false
 
 StackAlignment: 16
 
-MaximumGPRsPerAggregateArgument: 0
+MaximumGPRsPerAggregateArgument: 1
```

Comment:
These were quite broken, as it turns out. Good thing testing exists.

https://github.com/revng/revng/pull/232/commits/8402c0896bb7b98b5485ab56fca51f42e65586ad

@aleclearmind, feel free to close this if the commit looks good.

Reply:
the link was broken, it was intended to be https://github.com/revng/revng/pull/232/commits/18815ae21e799a50db947d974a76a0530b2a3b4b.

the commit is `ABI: fix issues with regparm abi definitions`.